src/sections/Community/Handbook/repo-data.js

```diff
@@ -168,24 +173,28 @@ export const repo_data = [
         project: "Cloud Native Labs",
```

Comment:
Please change this to Cloud Native Learning Paths (https://layer5.io/learn/learning-paths)